### PR TITLE
Update JenkinsfileCodeReview

### PR DESCRIPTION
**Pull request by edgibbs**
Setting up with newly created state Jenkins

### CODE REVIEW

**Comment by edgibbs** [comment]
Just testing build trigger process.

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

File: JenkinsfileCodeReview [hunk 1]
```diff
@@ -13,10 +13,11 @@ node {
       app.withRun("-e CI=true") { container ->
         stage('Lint') {
           sh "docker exec -t ${container.id} yarn lint"
+          sh "docker exec -t ${container.id} bundler-audit"
+          sh "docker exec -t ${container.id} brakeman"
         }
         stage('Unit Test') {
-          sh "docker exec -t ${container.id} bundle exec rspec"
-          sh "docker exec -t ${container.id} yarn run test"
+          sh "docker exec -t ${container.id} yarn test"
```
Comment (eckdanny-osi) [inline JenkinsfileCodeReview]: ahh Good!

This must've gotten overlooked in #123 (neglected multiple `Jenkinsfile`s)

**Pull request merged**